aidb_utilities/db_setup/clear_cache.py

```diff
@@ -0,0 +1,55 @@
+from collections import deque
+
+from sqlalchemy.schema import ForeignKeyConstraint
+from sqlalchemy.sql import delete
+
+from aidb.engine.engine import Engine
+from aidb.inference.bound_inference_service import CachedBoundInferenceService
+from aidb.utils.logger import logger
+
+
+async def clear_ML_cache(engine: Engine):
+  '''
+  Clear the cache table at start if the ML model has changed.
+  Delete the cache table for each service.
+  '''
+  for inference_binding in engine._config.inference_bindings:
```

Comment:
Can you refactor the code?
1. Collect all tables.
2. Construct a Table Graph
3. topological sort
4. Delete tables.
You can refer to https://github.com/ddkang/aidb/blob/d8e78e488ac91b78e8cc002b06aa01dfa8f39dfd/aidb/config/config.py#L65 and https://github.com/ddkang/aidb/blob/d8e78e488ac91b78e8cc002b06aa01dfa8f39dfd/aidb/config/config.py#L121C23-L121C39

Reply:
Ok, modified that.